piexif/_transplant.py

```diff
@@ -3,7 +3,7 @@
 from ._common import *
 
 
-def transplant(exif_src, image, new_file=None):
+def transplant(exif_src, image, new_file=None, include_iptc=False):
```

Comment:
I think it would deserve a test in https://github.com/hMatoba/Piexif/blob/master/tests/s_test.py#L439

Reply:
And also some explanation in https://github.com/hMatoba/Piexif/blob/master/doc/functions.rst